images/linux/scripts/SoftwareReport/SoftwareReport.CachedTools.psm1

```diff
@@ -28,6 +28,33 @@ function Get-ToolcacheGoVersions {
     return Get-ChildItem $toolcachePath -Name | Sort-Object { [Version]$_ }
 }
 
+function Build-GoEnvironmentTable {
+    Write-Host "Build-GoEnvironmentTable"
+    return @(
+        @{
+            "Name" = "GOROOT"
+            "Value" = $env:GOROOT
+        },
+        @{
+            "Name" = "GOROOT_1_13_X64"
+            "Value" = $env:GOROOT_1_13_X64
+        },
+        @{
+            "Name" = "GOROOT_1_14_X64"
+            "Value" = $env:GOROOT_1_14_X64
+        },
+        @{
+            "Name" = "GOROOT_1_15_X64"
+            "Value" = $env:GOROOT_1_15_X64
+        }
+    ) | ForEach-Object {
+        [PSCustomObject] @{
+            "Name" = $_.Name
+            "Value" = Get-PathWithLink($_.Value)
```

Comment:
These variables are not symlinks so we don't need to use `Get-PathWithLink` here